detox/test/e2e/09.stress-timeouts.test.js

```diff
@@ -4,7 +4,7 @@ describe('StressTimeouts', () => {
     await element(by.text('Timeouts')).tap();
   });
 
-  it('should handle a short timeout', async () => {
+  it(':ios: should handle a short timeout', async () => {
     await element(by.id('TimeoutShort')).tap();
```

Comment:
Why did you disable tests for Android?

Reply:
I'll try to run them from updated master and let you know the result